securedrop_client/gui/widgets.py

```diff
@@ -642,8 +646,20 @@ def __init__(
         self.empty_conversation_view = EmptyConversationView()
         self.view_layout.addWidget(self.empty_conversation_view)
 
-        # Add widgets to layout
-        self._layout.addWidget(self.source_list, stretch=1)
+        self.sources_pane_holder = QWidget()
+        # TODO - remove this hard-coded width
```

Comment:
is this possible to remove it?

Reply:
I couldn't figure out another way :/
Without that style, on wide screens the toolbar is wider that the source list and there's an ugly grey space.

![Screenshot 2023-02-08 at 11 50 30](https://user-images.githubusercontent.com/17057932/217522166-f04ab767-3425-419e-93a8-f8a45d7e2e91.png)

Reply:
no problem